code/modules/clothing/shoes/colour.dm

```diff
@@ -59,11 +59,6 @@
 	icon_state = "purple"
 	item_color = "purple"
 
-/obj/item/clothing/shoes/brown
-	name = "brown shoes"
-	icon_state = "brown"
-	item_color = "brown"
```

Comment:
can you readd the item color to the other brown definition

Reply:
brown shoes are subtyped higher in the file and I don't want compile jank to happen due to this

Reply:
I know, I just mean readd the item color variable to the brown shoes

Reply:
👍 readding